src/config/consts.js

```diff
@@ -1,4 +1,14 @@
+const ONCE_A_DAY = 'ONCE_A_DAY';
+
```

Comment:
without this `const ONCE_A_DAY = 'ONCE_A_DAY';` it is not possible to apply the same value to `ONCE_A_DAY`, `STATIC_CONTENT`, `NEWS`, `EVENTS`, `POINTS_OF_INTEREST_AND_TOURS`. we cannot reference a key as a value inside of `consts` and we do not want to hardcode the string `'ONCE_A_DAY'` to all of them. imagine once the string needs to change, then it would be multiple lines to change. with having that first line, only that needs to be changed.